torch/nn/_functions/padding.py

```diff
@@ -4,6 +4,17 @@
 class ConstantPadNd(Function):
 
     @staticmethod
+    def symbolic(g, input, pad, value=0):
+        dim = len(input.type().sizes())
+        paddings = []
+        for i, j in zip(pad[0::2], pad[1::2]):
```

Comment:
Can you add a comment saying what's the expected ONNX format for padding sizes?

Reply:
Sure.

Reply:
check that pad is divisible by 2? (zip just ignores stuff)

Reply:
This is already checked by padding op in pytorch. So we won't have odd pad. I will add comments mentioning this. :-)